apps/angular/pipe-easy/src/app/app.component.ts

```diff
@@ -1,21 +1,21 @@
-import { NgFor } from '@angular/common';
+import { NgFor, TitleCasePipe } from '@angular/common';
 import { Component } from '@angular/core';
+import { ComputationPipe } from './computation.pipe';
 
 @Component({
   standalone: true,
-  imports: [NgFor],
+  imports: [NgFor, ComputationPipe, TitleCasePipe],
```

Comment:
no more ngFor needed

Reply:
Removed ngFor from imports array.  I forgot to delete ngfor from there. New syntax of ngFor does not require to import into component imports array. That's cool thing.